governance/pyth_staking_sdk/src/staking/accounts.ts

```diff
@@ -0,0 +1,59 @@
+import { BorshCoder } from "@coral-xyz/anchor";
+import { PublicKey } from "@solana/web3.js";
+import type { Position, PositionAnchor } from "../types";
+import type { Staking } from "../../types/staking";
+import { POSITION_BUFFER_SIZE, POSITIONS_ACCOUNT_SIZE } from "../constants";
+import { convertBNToBigInt } from "../utils/bn";
+
+export type StakeAccountPositions = {
+  address: PublicKey;
+  data: {
+    owner: PublicKey;
+    positions: (Position | null)[];
+  };
+};
+
+export class StakeAccountPositionsAnchor {
+  public address: PublicKey;
+  public data: PositionAccountDataAnchor;
+
+  constructor(address: PublicKey, data: Buffer, idl: Staking) {
```

Comment:
This problem was already in the old code, but isn't it weird that the idl is an argument, when we could import the idl in this file?

Reply:
that was my initial thought too, but when importing the idl it doesn't work for some reason.